core/domain/blog_services.py

```diff
@@ -31,8 +31,8 @@
 from core.domain import user_domain
 from core.platform import models
 
-from typing import Callable, List, Optional, Sequence, Tuple, overload
-from typing_extensions import Literal, TypedDict
+from typing import (
+        Callable, List, Literal, Optional, Sequence, Tuple, TypedDict, overload)
```

Comment:
Shouldn't this be indented by 4 not 8? Ditto in a bunch of other files.

Reply:
yeah sure ! There a couple other files where i need to change the imports from typing_extensions to typing. Will do it.